docker/windows/start.ps1

```diff
@@ -54,9 +54,6 @@ if (-not (Test-Path Env:AZP_URL)) {
       --work "$(if (Test-Path Env:AZP_WORK) { ${Env:AZP_WORK} } else { '_work' })" `
       --replace
   
-    # remove the administrative token before accepting work
-    Remove-Item $Env:AZP_TOKEN_FILE
-  
```

Comment:
Why are you removing this? How it relates to the issue this PR is trying to solve?

Reply:
if you read line 67 of this file which is used to clean up the agent instance in the pool it needs this file and if we delete before that step the cleanup fails.